platforms/documentation/docs/src/docs/userguide/reference/gradle_daemon.adoc

```diff
@@ -212,40 +215,111 @@ The following file will be generated:
 
 ----
 $ ./gradlew updateDaemonJvm --jvm-version=17
 ----
 
-If you run the task without any arguments, and the properties file does not exist, then the version of the current JVM used by the daemon will be used.
-
-NOTE: Gradle only supports the major JVM version and JVM vendor as a criterion.
-Support for other criteria may be added in a future release.
+If you run the task without any arguments, and the properties file does not exist, then the version of the current JVM used by the daemon will be used and the file will be created.
 
 On the next execution of the build, the Gradle client will use this file to locate a compatible JVM installation and start the daemon with it.
 
+NOTE: Gradle currently supports only the major JVM version (`--jvm-version`) and JVM vendor (`--jvm-vendor`) as criteria. Support for additional criteria may be added in a future release.
+
 ==== Specifying a JVM vendor
 
-Like the JVM version, the JVM vendor can be used as criteria to select a compatible JVM installation for the build. When no JVM vendor is specified, Gradle will consider all vendors compatible.
+The JVM vendor, like the JVM version, can be used as a criterion to select a compatible JVM installation for the build.
+If no vendor is specified, Gradle considers all vendors compatible.
+
+By default, running `updateDaemonJvm` to create the `gradle-daemon-jvm.properties` file will not generate a JVM vendor criterion.
+To specify a vendor, either configure it in the build script or pass it on the command line using `--jvm-vendor=<value>`:
+
+----
+$ ./gradlew updateDaemonJvm --jvm-version=17 --jvm-vendor=adoptium
+----
 
-By default, running `updateDaemonJvm` to create the `gradle-daemon-jvm.properties` file will not generate a JVM vendor criterion. You must either explicitly specify a JVM vendor for the `updateDaemonJvm` task in the build script or pass a JVM vendor on the command-line with `--jvm-vendor=<value>`.
+Gradle recognizes some JVM vendor strings as equivalent, such as "Adoptium" and "Temurin."
+You can view the list of recognized vendors by running `gradle help --task updateDaemonJvm`.
 
-Gradle recognizes a small number of JVM vendor strings as special and equivalent. For example, "Adoptium" and "Temurin" are considered the same vendor. You can see the list of special vendors by running `gradle help --task updateDaemonJvm`.
+If the specified vendor is not one of the recognized equivalents, Gradle will match it exactly.
+For example, "My Custom JVM" would require an exact match of the vendor name.
 
-If the JVM vendor you specify is not treated as a special value, Gradle considers the value as an exact match. For example, to match the vendor "My Custom JVM", the vendor criterion must be "My Custom JVM".
+The vendor can be configured on the task using <<toolchains.adoc#sec:vendors,the same syntax as the Java toolchain spec>>.
 
+[[sec:daemon_jvm_discovery]]
```

Comment:
Probably some reference that Daemon JVM discovery can be disabled in the way as toolchain task https://docs.gradle.org/current/userguide/toolchains.html#sub:disable_auto_detect by using `org.gradle.java.installations.auto-detect` gradle property

Reply:
Fixed.